### ISSUE TITLE
Feedback for “Next.js Image”

### ISSUE THREAD

**Issue by JonathanX64** [shuding/nextra#2391]
Hello!

At the moment, when using native Markdown image syntax, the image gets wrapped into Next.js Image container, gets optimized on Vercel and served as it should be.

However, when using HTML syntax inside of Markdown, such as <img src="icon.png"/>, the image is being delivered directly using icon.png asset, without optimization.

HTML in Markdown is very useful, as it allows to apply more attributes to them (height, width, etc.), apply specific css to them, and so much more.

Please kindly enable Next.js Image wrapping for <img> images. Thanks ever so much.

**Comment by B2o5T** [comment]
https://nextra.site/docs/guide/advanced/table#why-this-happens

**Comment by JonathanX64** [comment]
Okay. Please kindly explain to me, can I somehow work around this limitation? If I use curly braces around div with img tags, will these imgs become optimised / rendered by Next.js? Or by enabling remark-mdx-disable-explicit-jsx package?

I'm completely new to web development, so all of this is a bit new to me.

Thanks.

**Comment by B2o5T** [comment]
It’s explained at the bottom of doc, use remark-mdx-disable-explicit-jsx

**Comment by JonathanX64** [comment]
I've added the remark-mdx-disable-explicit-jsx component, and configured it correctly (i.e. it works on tables), but it does not work with images, even though I've disabled the whitelist of components in the config.

The curly braces thing also does not work. No matter what, `<img src="icon.png" width="500" class="responsive"/>` results in an unoptimized image.

What else could be the fix? 

The file is .md, not .mdx, if that matters (and I'd rather keep it that way, because all Markdown editors are unfriendly to .mdx documents for some reason).

Thanks.
